### ISSUE TITLE
[docs] background notifications on iOS: missing documentation on BGTaskSchedulerPermittedIdentifiers

### ISSUE THREAD

**Issue by daankennes** [expo/expo#32396]
### Summary

The docs on background notifications for iOS instruct to do the following:

_In the array under the ios.infoPlist.UIBackgroundModes key in your [app config](https://docs.expo.dev/workflow/configuration), add the values remote-notification and processing._

I did this, but after trying to upload a production build to App Store Connect, I get this error:
`Missing Info.plist value. The Info.plist key 'BGTaskSchedulerPermittedIdentifiers' must contain a list of identifiers used to submit and handle tasks when 'UIBackgroundModes' has a value of 'processing'.`

There is no documentation on what should be done to prevent this error.

### Link to the related docs page

https://docs.expo.dev/versions/latest/sdk/notifications/#background-notifications

**Comment by chrfalch** [comment]
Hi @daankennes - Thanks for reporting this issue. The key `BGTaskSchedulerPermittedIdentifiers` is related to the new `BGTaskScheduler` API on iOS and not remote notifications. 

I'm wondering if you have additional dependencies or have added `process` to the `UIBackgroundModes` key in your Info.plist file which can cause such an error message from iTunes?

**Comment by daankennes** [comment]
Hi @chrfalch, yes that's true. I linked to the documentation on background notifications because in there it says to add `process` to `UIBackgroundModes`, but excludes further information on `BGTaskSchedulerPermittedIdentifiers`. There also no information on that key in the documentation on [TaskManager](https://docs.expo.dev/versions/latest/sdk/task-manager/).

**Comment by chrfalch** [comment]
Yeah, you shouldn't need the `process` key since Task Manger is not yet using it (it will in a later release) - so try removing `process` and see if that solves it - and ping back here with your findings :)

I'll make an update to the docs to clarify this.
